recipes/server.rb

```diff
@@ -11,7 +11,7 @@
 #    http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
+# distributed under the License is distributed on an "AS IS" BASIS
```

Comment:
We shouldn't be modifying the license header like this, please revert.

Reply:
i think this must have been an accident or cookstyle, sorry